rootex/framework/ecs_factory.cpp

```diff
@@ -141,6 +142,7 @@ void ECSFactory::Initialize()
 	ASSIGN_COMPONENT_SET(SkyComponent);
 	ASSIGN_COMPONENT_SET(CPUParticlesComponent);
 	ASSIGN_COMPONENT_SET(ParticleEffectComponent);
+	ASSIGN_COMPONENT_SET(SpringArmComponent);
```

Comment:
Put this one after line 117 instead. All components in the same category are listed together without spaces in between.